test/pylons_wallet/pylons_wallet_impl_test.dart

```diff
@@ -50,16 +50,14 @@ void goToInstallTest() {
 }
 
 void goToLoginTest() {
-  test(
-      'should simply redirect to the Pylons Wallet app',
-          () async {
-        mockChannelHandler();
-        var uniLink = MockUniLinksPlatform();
-        when(uniLink.linkStream)
-            .thenAnswer((realInvocation) => Stream<String?>.value('Corey'));
-        var pylonsWallet = PylonsWalletImpl(host: MOCK_HOST, uniLink: uniLink);
-        pylonsWallet.goToPylons();
-      });
+  test('should simply redirect to the Pylons Wallet app', () async {
+    mockChannelHandler();
+    var uniLink = MockUniLinksPlatform();
+    when(uniLink.linkStream)
+        .thenAnswer((realInvocation) => Stream<String?>.value('Corey'));
+    var pylonsWallet = PylonsWalletImpl(host: MOCK_HOST, uniLink: uniLink);
+    pylonsWallet.goToPylons();
```

Comment:
What are you expecting in this test ? With no expectation i will consider this test a useless test.

Reply:
@kjawadDeveloper2 I'd want to ask you what can be expected in this case really, as the `goToPylons()` does return void. I can't find any tests for void functions here in `pylons_wallet_impl_test.dart`.
With no expectation, considering it a useless test.... makes much sense. Can I just skip this test? I think `goToInstallTest()` also has the same issue, which had been added as per your request too. Please advise whenever you get some free time. thx.

Reply:
So basically we are redirecting it to the appstore or playstore . The package we are using in launchurl. You have to expect that when you call this method. The url that is received by this package is the same as you want. 
For example:
In case of android it should be the url of the playstore. 
In case of ios it should be the url of the appstore.

Reply:
@kjawadDeveloper2 , please review again when you are available. I've made some changes and added the `expect` in test codes.